### ISSUE TITLE
SqlException when downgrading from Rhetos 4.0.0 to an older version

### ISSUE THREAD

**Issue by bantolov** [Rhetos/Rhetos#353]
When downgrading from Rhetos 4.0.0 to an older version, DeployPackages results with the following error:

```log
FrameworkException: SqlException has occurred:
Msg 4902, Level 16, State 1, Line 100
Cannot find the object "Rhetos.DatabaseGeneratorAppliedConcept" because it does not exist or you do not have permissions.
```

**Comment by bantolov** [comment]
This issue is caused by incorrect conditions in database setup script RhetosDatabase.MsSql.sql in older versions of Rhetos (1.0.0 - 3.0.1).
It is fixed in v3.1.0, but this fix does not help when downgrading to an older version.

**Comment by bantolov** [comment]
Fixed in v4.0.1.
